go/libraries/doltcore/sqle/database.go

```diff
@@ -183,7 +193,7 @@ func (db *Database) DropTable(ctx *sql.Context, tableName string) error {
 // CreateTable creates a table with the name and schema given.
 func (db *Database) CreateTable(ctx *sql.Context, tableName string, schema sql.Schema) error {
 
-	if !doltdb.IsValidTableName(tableName) {
+	if !doltdb.IsValidTableName(tableName) || tableName == doltdb.DocTableName {
```

Comment:
I would just check strings.HasPrefix(tableName, "dolt_"). We want to reserve that entire namespace. Also make "dolt_" a constant somewhere in this file.

Reply:
Resolved: https://github.com/liquidata-inc/dolt/pull/280/commits/fc2e09aad5f6b2267c341262635a0aa48b76b2cf#diff-ff4ec6598b401e530eb84ad72ea62a9fR165